pages/common/hadolint.md

```diff
@@ -0,0 +1,16 @@
+# hadolint
+
+> Lints Dockerfiles to help you build [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) Docker images.
+> More information: https://github.com/hadolint/hadolint.
+
+- Lint a Dockerfile, printing any output to STDOUT:
```

Comment:
the `eslint` doesn't make any reference to `print to stdout`. I prefer to remove that
```suggestion
- Lint a Dockerfile:
```

Reply:
Otherwise, we use this format for `stdout`:
```suggestion
- Lint a Dockerfile, printing the output to `stdout`:
```

Reply:
I don't think `stdout` is particularly important here